frontend/src/store/FamLoginUserState.ts

```diff
@@ -106,6 +113,20 @@ const getApplicationsUserAdministers = () => {
     return applicationList;
 };
 
+const isApplicationAdmin = () => {
+    const appsUserIsAdmin = getUserAccess().filter(
+        (access) => access.auth_key == APP_ADMIN_ROLE
+    );
+
+    if (appsUserIsAdmin.length > 0) {
+        appsUserIsAdmin[0].grants.filter(
+            (grant) => grant.application.id == FAM_APPLICATION_ID
```

Comment:
Only for some certain scenarios, otherwise please do not rely on the hardcoded FAM_APPLICATION_ID = 1.
Also, is this the grant on APP_ADMIN? I don't think you will find FAM application at APP_ADMIN grant.

Reply:
oh yes, it should be the selected application id

Reply:
Already made the change to selectedApplicationId